packages/trader/src/Modules/CFD/Containers/cfd-reset-password-modal.jsx

```diff
@@ -16,9 +16,10 @@ import { localize, Localize, getLanguage } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import { getMtCompanies } from 'Stores/Modules/CFD/Helpers/cfd-config';
 
+const reset_password_type = localStorage.getItem('cfd_reset_password_type') || 'main'; // Default to main
```

Comment:
If we move this line outside the function, it's gonna get the `cfd_reset_password_type` only once. It won't be able to get any new values set to `cfd_reset_password_type`. Is that fine?